posts/2022-07-25-keyword-generics.md

```diff
@@ -0,0 +1,344 @@
+---
+
+layout: post
+title: "Announcing the Keyword Generics Initiative"
+author: Yoshua Wuyts, on behalf of the Keyword Generics Initiative
+release: false
+---
+
+We ([Oli], [Niko], and [Yosh]) are excited to announce the start of the [Keyword
+Generics Initiative][kwi], a new initiative [^initiative] under the purview of
+the language team. We're officially just a few weeks old now, and in this post
+we want to briefly share why we've started this initiative, and share some
+insight on what we're about.
+
+[Oli]: https://github.com/oli-obk
+[Niko]: https://github.com/nikomatsakis
+[Yosh]: https://github.com/yoshuawuyts
+[kwi]: https://github.com/rust-lang/keyword-generics-initiative
+
+[^initiative]: Rust governance terminology can sometimes get confusing. An
+"initiative" in Rust parlance is different from a "working group" or "team".
+Initiatives are intentionally limited: they exist to explore, design, and
+implement specific pieces of work - and once that work comes to a close, the
+initiative will wind back down. This is different from, say, the lang team -
```

Comment:
I think it's probably worth explicitly noting somewhere in this post that initiatives are really intended as explore/design space - there's not actually any guarantees or even strong confidence at this early stage in the process that any of the particulars will be adopted by Rust. Many of our initiatives come to fruition in somewhat different forms or not at all, particularly those relatively early in the process.

Might make for a good FAQ question (something like "Q: Where is the RFC?", perhaps?).

Reply:
Good call; done!